### ISSUE TITLE
xml-apis:xml-apis required for AGP 4.2

### ISSUE THREAD

**Issue by nrobi144** [icerockdev/moko-resources#120]
After updating  to Android Studio 4.2 Canary 14 and associated gradle update I started getting following build error:

```
Execution failed for task ':common:generateMRcommonMain'.
> org/apache/batik/w3c/dom/ElementTraversal
```

Adding `xml-apis:xml-apis:1.4.01` to the top of the buildscript dependencies fixes the problem.
The issue looks similar to https://github.com/cashapp/sqldelight/issues/2058

**Comment by Alex009** [comment]
thanks for report. i think android gradle plugin team will fix dependency problem before 4.2 plugin release...but if not - we will add dependency by self.

**Comment by nrobi144** [comment]
Seems to be fixed with `AGP 7.0.0-alpha02`

**Comment by hardysim** [comment]
This happens with `AGP 4.2.0-beta03` as well. Adding `xml-apis` as a workaround does the trick. I hope there will be a fix for AGP 4.2 as well.

**Comment by Alex009** [comment]
actual release (0.18.0) use AGP 7 and all ok. so issue not actual anymore
